internal/mesh/internal/controllers/sidecarproxy/builder/destination_multiport_test.go

```diff
@@ -41,6 +41,31 @@ func TestBuildMultiportImplicitDestinations(t *testing.T) {
 		},
 	}
 
+	multiportServiceData := &pbcatalog.Service{
```

Comment:
previously this test used a shared version of this type of data with other tests and it was causing lots of clobbering and coordination.  opted to make this local to the test.